src/merge_bot/merge_bot.py

```diff
@@ -158,6 +158,33 @@ def grab_go_version_from_go_mod():
     return ""
 
 
+# Run `make merge-bot` to allow projects to run custom logic when merging
+# upstream changes. This is useful for projects that need to update vendored
+# dependencies or run other tasks after an upstream merge.
+# This is a best effort, and if the target doesn't exist, we ignore the error.
+def run_make(repo):
+    try:
+        proc = subprocess.run(
+            "make merge-bot", shell=True, check=True, capture_output=True
+        )
+        logging.debug(f"make merge-bot output: {proc.stdout.decode()}")
+    except subprocess.CalledProcessError as err:
+        if "No rule to make target" not in err.stderr.decode():
+            raise RepoException(
```

Comment:
Should we really raise a `RepoException` when the repository doesn't have a `make merge-bot` target, and trigger a `Manual intervention is needed to merge {source} into {dest}` message on slack? IMO, we should log what happens but not send update to slack, so perhaps raise a different exception type.

Reply:
that was a bad copy paste.

Reply:
* If the repo doesn't have `merge-bot` target, just move on: no error at all.
* If the repo has the target but running it failed, we send an error but we also send an update on Slack saying that manual intervention is needed in the PR. I think it's fine this way.